aws-api/src/androidTest/java/com/amplifyframework/api/aws/RestApiInstrumentationTest.java

```diff
@@ -15,37 +15,63 @@
 
 package com.amplifyframework.api.aws;
 
-import com.amplifyframework.AmplifyException;
+import android.util.Log;
+
 import com.amplifyframework.api.rest.RestOptions;
 import com.amplifyframework.api.rest.RestResponse;
 import com.amplifyframework.core.Amplify;
 
+import com.amazonaws.mobile.client.AWSMobileClient;
+import com.amazonaws.mobile.client.Callback;
+import com.amazonaws.mobile.client.UserStateDetails;
 import org.json.JSONException;
 import org.json.JSONObject;
 import org.junit.BeforeClass;
 import org.junit.Test;
 
+import java.util.concurrent.CountDownLatch;
+
+import static androidx.test.core.app.ApplicationProvider.getApplicationContext;
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
 
 /**
  * Validates the functionality of the {@link AWSApiPlugin} for REST operations.
- *
  */
 public final class RestApiInstrumentationTest {
 
     /**
      * Configure the Amplify framework, if that hasn't already happened in this process instance.
-     * @throws AmplifyException Exception is thrown if configuration fails.
+     *
+     * @throws Exception Exception is thrown if configuration fails.
      */
     @BeforeClass
-    public static void onceBeforeTests() throws AmplifyException {
+    public static void onceBeforeTests() throws Exception {
         AmplifyTestConfigurator.configureIfNotConfigured();
+
+        final CountDownLatch latch = new CountDownLatch(1);
+        AWSMobileClient.getInstance().initialize(getApplicationContext(), new Callback<UserStateDetails>() {
```

Comment:
Would be beneficial to move this to a core test configurator with the pattern of latched response listener so we can reuse this for all other categories.

Reply:
Will handle this as part of another PR

Reply:
I updated the current PR to include this change